Estaffieri/romcom
# User cover display is working

### What's this PR do?
This should take what is input into the make new cover page and display that cover.
### Where should the reviewer start?
main.js 
### How should this be manually tested?
Navigate to make new cover button, fill out form and see that the new cover displays while form view is hidden.
